content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -443,6 +444,18 @@ Watching... /home/prismauser/prisma/prisma-play/prisma/schema.prisma
 
 </CodeWithResult>
 
+##### Run the `generate` command with a specific generator
```

Comment:
Weird grammar a bit, but maybe more precise?
```suggestion
##### Run the `generate` command with only a specific generator
```

Reply:
Ah yes, that's more concise. Thanks 👍🏻